src/View/base.html.twig

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+
+<html>
+
+  <head>
+
+    <meta charset="UTF-8">
+    <title>
+        {% block title %}
+        Quizzy
+        {% endblock %}
+    </title>
+
+    {% block stylesheets %}
+    <link rel = "stylesheet" href = "mainStyle.css">
+    {% endblock %}
+
+  </head>
+
+  <body>
+
+    <header>
+        <h1>Quizzy</h1>
+    </header>
+
+    {% block content %}
+    {% endblock %}
+
+    {% block scripts %}
+    {% endblock %}
```

Comment:
If the idea of the script block is to add javascripts, it probably should be at the end of the body : after the footer

Reply:
updated